peachjam/views/generic_views.py

```diff
@@ -168,6 +168,9 @@ def get_context_data(self, **kwargs):
         )
         context["labels"] = doc.labels.all()
 
+        if doc.doc_type == "judgment":
+            context["judges"] = self.add_judges(doc)
```

Comment:
Naming: all the other `add_xxx` methods take the context and add something to the context, hence the "add" in the name. This one doesn't add anything, it just returns something. So from a readability standpoint, either `add_judges` should be `get_judges`, or it should take `context` and update the context.

Additionally, rather put this into `views.judgment.JudgmentDetailView` since it only applies to judgments, then you don't need to check the `doc_type`.